### PR TITLE
Add Kirby Fighters Deluxe

### PR DESCRIPTION
**Pull request by Sedhaild**
Adds Kirby Fighters Deluxe.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/kirby-fighters-deluxe/game.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title = "Kirby Fighters Deluxe"
+description = "Kirby Fighters Deluxe is a battle-action game developed by HAL Laboratory, where the player can use Kirby's Copy Abilities. It is an upgraded version of the Kirby: Triple Deluxe sub-game, Kirby Fighters. It has improved single- and multi-player modes, as well as new items and stages."
```
Comment (Leo626) [inline games/kirby-fighters-deluxe/game.dat]: The hyphen in `single-` doesn't look right. I think it should be just be `single`.

**Comment by Leo626** [comment]
The wiki should also describe the graphics and audio (accuracy of the emulation) as well. Other than that and my small nitpick in the description, LGTM.

**Comment by Sedhaild** [comment]
I'm sorry, I'm a very new with all this github thing (but I'm learning) and I can't find the wiki I made for this game to edit it.

**Comment by Leo626** [comment]
To the top right is the Edit button: https://github.com/citra-emu/citra-games-wiki/wiki/kirby-fighters-deluxe.

**Pull request merged**